tests/test_operation.py

```diff
@@ -2111,31 +2111,60 @@ class TestExpandMatrix:
     base_matrix_1_broadcasted = np.arange(1, 13).reshape((3, 2, 2))
     base_matrix_2 = np.arange(1, 17).reshape((4, 4))
     base_matrix_2_broadcasted = np.arange(1, 49).reshape((3, 4, 4))
+    base_matrix_3 = np.arange(1, 10).reshape((3, 3))
+    base_matrix_3_broadcasted = np.arange(1, 19).reshape((2, 3, 3))
+    base_matrix_4 = np.arange(1, 82).reshape((9, 9))
```

Comment:
I chose not to use a broadcasted version of that matrix because creating an "expected" output matrix for that case would've been extremely tedious. Instead, I used matrices from `tests/gate_data.py` which I already had working permutations of.